app/models/user_column.rb

```diff
@@ -0,0 +1,12 @@
+class UserColumn < ApplicationRecord
+  include Exportable
+  extend FriendlyId
+  friendly_id :name
+  include Parameterizable::ByIdName
+
+  belongs_to :user
+  validates :user_id, :name, :presence => true
+  serialize :columns
+  attr_exportable :name, :columns
+  scope :find_by_name, ->(name) { where(:name => name)}
```

Comment:
Isn't this provided by rails? Iirc it is and it returns single record. Why do you define such scope? It's equivalent to `.where(:name => name)`